### ISSUE TITLE
stats: TestAtMostOneRunningCreateStats sometimes fails when run against test tenant

### ISSUE THREAD

**Issue by yuzefovich** [cockroachdb/cockroach#109379]
As title, sometimes it succeeds, sometimes it fails and then it's slow to quiesce.

Jira issue: CRDB-30891
Epic CRDB-48945

**Comment by mgartner** [comment]
Now that #108762 is merged, can this issue be closed?

**Comment by yuzefovich** [comment]
No, the issue is still current - `TestAtMostOneRunningCreateStats` explicitly disables the test tenant referencing this issue.
